src/sonic-config-engine/sonic-cfggen

```diff
@@ -271,7 +275,11 @@ def main():
         deep_update(data, json.loads(args.additional_data))
 
     if args.from_db:
-        configdb = ConfigDBConnector(**db_kwargs)
+        if args.namespace is None:
+            configdb = ConfigDBConnector(**db_kwargs)
+        else:
+            configdb = ConfigDBConnector(use_unix_socket_path=True, namespace=args.namespace, **db_kwargs)
```

Comment:
make sure for namespace cases the application always  has root access, otherwise it will fail.

Reply:
Yes, we need to make sure that the application has sudo access , if need to access the DB in another namespace using sonic-db-cli. This should be added in the the script where ever we invoke the sonic-cfggen/sonic-db-cli.